README.md

```diff
@@ -2,15 +2,75 @@
 
 ## Intro
 
-This repository contains the backend logic for the dfusion exchange based on the specification, see [here](github.com/gnosis/dex-research)
+This repository contains the backend logic for the dfusion exchange based on [this specification](github.com/gnosis/dex-research).
 
-## Architecture
+It contains two sub-projects that both implement the market mechanism described above in different ways. An fully on-chain solution with instant finality but limited scalability (referred to as "StableX") and a preliminary version that intends to achieves scalability by offloading computation and data-storage off-chain using an [optimistic roll-up](https://medium.com/plasma-group/ethereum-smart-contracts-in-l2-optimistic-rollup-2c1cef2ec537) approach. The latter is in early development stage and not yet ready for use.
 
-<p align="center">
-  <img src="documentation/architecture.png" alt="dex-services architecture" width="500">
```

Comment:
I was also wondering if we might remove this diagram soon. No worries! It will return (eventually)